packages/fluentui/react-northstar/src/components/Pill/Pill.tsx

```diff
@@ -0,0 +1,112 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import { UIComponentProps, ContentComponentProps, commonPropTypes, SizeValue, createShorthand } from '../../utils';
+import { ShorthandValue, FluentComponentStaticProps } from '../../types';
+import { BoxProps } from '../Box/Box';
+
+import {
+  ComponentWithAs,
+  useAccessibility,
+  getElementType,
+  useStyles,
+  useTelemetry,
+  useFluentContext,
+  useUnhandledProps,
+} from '@fluentui/react-bindings';
+import { PillContent } from './PillContent';
+
+export interface PillProps extends UIComponentProps, ContentComponentProps<ShorthandValue<BoxProps>> {
+  /**
+   * A Pill can be sized.
+   */
+  size?: Extract<SizeValue, 'smaller' | 'small' | 'medium'>;
+
+  /**
+   * A Pill can be rounded
+   */
+  rounded?: boolean;
+
+  /**
+   * A Pill can be filled, inverted or outline
+   */
+  appearance?: 'filled' | 'inverted' | 'outline';
+}
+
+export type PillStylesProps = Required<Pick<PillProps, 'appearance' | 'size' | 'rounded'>>;
+
+export const pillClassName = 'ui-pill';
+
+/**
+ * THIS COMPONENT IS UNSTABLE
```

Comment:
Does this mean that the component is still in development and should not be used in production ?

Reply:
Yes

Reply:
Let's do this, `unstable` is a nice short way to do it for props, but nicer to be more explicit here IMO

```suggestion
 * THIS COMPONENT IS STILL IN DEVELOPMENT AND IS NOT READY FOR PRODUCTION
```